frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```
@@ -443,8 +443,9 @@
   "Enable arbiter": "Enable arbiter",
   "If not labeled, the selected nodes are labeled <1>{{label}}</1> to make them target hosts for OpenShift Container Storage&apos;s components.": "If not labeled, the selected nodes are labeled <1>{{label}}</1> to make them target hosts for OpenShift Container Storage&apos;s components.",
   "selected ({{cpu}} CPU and {{memory}} on ": "selected ({{cpu}} CPU and {{memory}} on ",
-  "Enable taint nodes": "Enable taint nodes",
-  "Selected nodes will be dedicated to OpenShift Container Storage's use only": "Selected nodes will be dedicated to OpenShift Container Storage's use only",
+  "Mark nodes as dedicated": "Mark nodes as dedicated",
+  "This will taint the nodes with the<1>key: node.ocs.openshift.io/storage</1>, <3>value: true</3>, and <6>effect: NoSchedule</6>": "This will taint the nodes with the<1>key: node.ocs.openshift.io/storage</1>, <3>value: true</3>, and <6>effect: NoSchedule</6>",
+  "Selected nodes will be dedicated to OCS use only": "Selected nodes will be dedicated to OCS use only",
```

We are asked to avoid OCS abbrev in UI.
```suggestion
  "Selected nodes will be dedicated to OCS use only": "Selected nodes will be dedicated to OpenShift Container Storage use only",
```

---

No clue how this got in here, reverting.